gensim/summarization/textcleaner.py

```diff
@@ -8,16 +8,14 @@
 from gensim.utils import tokenize
 from six.moves import xrange
 import re
-import logging
+import warnings
 
-logger = logging.getLogger('summa.preprocessing.cleaner')
 
 try:
     from pattern.en import tag
-    logger.info("'pattern' package found; tag filters are available for English")
```

Comment:
Please keep this line. Info is used for confirmation that things are working as expected.